csupport/addC.go

```diff
@@ -0,0 +1,101 @@
+package csupport
+
+import (
+	"canaveral/lib"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// createMainFile creates a basic projName.c file that uses stdlib.h
+// to print "Canaveral is awesome!" in the main function
+// * tested
+func createMainFile(projName string) error {
+	err := lib.CreateFile(projName + ".c")
+	if err != nil {
+		return err
+	}
+	mainfile, err := os.OpenFile(projName+".c", os.O_APPEND|os.O_WRONLY, 0644)
+	defer func() {
+		err := mainfile.Close()
+		lib.Check(err)
+	}()
+
+	include := "#include <stdio.h>"
+	main := "int main() {\n\tprintf(\"Canaveral is awesome!" + `\n` + "\");\n\treturn 0;\n}\n"
+
+	_, err = mainfile.WriteString(include + "\n\n" + main)
+	return err
+}
+
+// createREADME creates a README.md file
+// * tested
+func createREADME(projName string) error {
```

Comment:
Wait, this create readme function is very useful.  I don't recall if the python one has readme creation. If it doesn't, can you do something similar for that too?

This is awesome

Reply:
Yep it does. I have been thinking about making it a library function that takes in a project name, the body text, and always appends "Created by Canaveral" to the end. Thoughts?

Reply:
I like that. Do it